include/filewriter/summarywriter_impl.hpp

```diff
@@ -23,6 +23,33 @@ void SummaryWriter<Filewriter>::Scalar(const std::string& tag,
     fw.CreateEvent(step, summary);
 }
 
+template<typename Filewriter>
+void SummaryWriter<Filewriter>::Text(const std::string &tag,
+                                     int step,
+                                     const std::string& text,
+                                     Filewriter& fw)
+{
+    mlboard::SummaryMetadata_PluginData *pluginData =
```

Comment:
I think you should use two spaces (one tab) instead of four spaces. Indentation error.